sys/cam/ctl/ctl_cmd_table.c

```diff
@@ -1227,7 +1227,7 @@ const struct ctl_cmd_entry ctl_cmd_table[256] =
 					  CTL_CMD_FLAG_ALLOW_ON_PR_RESV |
 					  CTL_FLAG_DATA_IN,
  CTL_LUN_PAT_NONE,
- 10, {0x02, 0x01, 0, 0, 0, 0xff, 0xff, 0xff, 0x07}},
+ 10, {0xe1, 0, 0, 0x54, 0, 0x0, 0xff, 0xff, 0xc7}},
```

Comment:
OK... MMC4 says this mask should be:
{ 0x01, 0, 0, 0xff, 0, 0, 0xff, 0xff, ??? } (I couldn't find control byte in mmc and didn't want to go hunting)
MMC3 -- same.
MMC2 -- same.
MMC -- not present

So that's somewhat different... Though MMC2 does have the upper three bits separated from the rest, so that kinda makes sense. I had trouble snagging your document

Reply:
I haven't looked deep yet, but just for a note, here should be not simply the bits defined in specs, but only the bits supported by the ctl_get_event_status() function.  I see it checks only one bit from the byte2.  We should check that no other bits left allowed but not handled do anything wrong.

Reply:
I agree with Warner's version.

Reply:
Okay, I'll adjust the PR